cmd/qgb/query/cmd.go

```diff
@@ -60,7 +60,7 @@ func Signers() *cobra.Command {
 			ctx, cancel := context.WithCancel(cmd.Context())
 			defer cancel()
 
-			stopFuncs := make([]func() error, 0)
+			stopFuncs := make([]func() error, 1)
```

Comment:
yes, also, because if you specify  the length of the slice to 1, if we run `append(...)`, it will put the stop function as the second element while the first element being empty. In that case, what we can do is instead of using `append(...)`, we use `stopFuncs[0]=...`. If not, we will end up with a slice of 2 elements. https://go.dev/play/p/xwxwBHFtpGF?v=